src/app/replicator/transformer/cozy_to_android_contact.coffee

```diff
@@ -69,10 +69,14 @@ _tags2Categories = (tags) ->
 # Build pohto (list) field from contact's photo.
 _attachments2Photos = (contact) ->
     if contact._attachments? and 'picture' of contact._attachments
-        photo = new ContactField 'base64', contact._attachments.picture.data
-
-        return [photo]
-
+        try
+            data = contact._attachments.picture.data
+            window.atob data
```

Comment:
@kosssi I don't understand these 2 lines. It looks like the result of `atob` is not used later, so I don't see what is the purpose of these 2 lines.